CenterForOpenScience/ember-osf-web
# [ENG-4011]  Fix deprecation - implicit injections

-   Ticket: [ENG-4011]

## Purpose

To fix the deprecation warning for implicit injections

## Summary of Changes

1. Remove deprecation silencing
2. Upgrade ember-metrics away from our branch to a modern version
3. Remove unused keen metrics adapter

## Side Effects

The only likely thing to break if something went wrong is Google Analytics. 

## QA Notes

Verify that clicking on buttons still generates 'collect' network requests, which are the ones that go to Google Analytics. These should have information on them such as if we're logged in, and where the event was based on in the structure of the site.


[ENG-4011]: https://openscience.atlassian.net/browse/ENG-4011?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
